packages/contracts/test/app-registry.spec.ts

```diff
@@ -0,0 +1,230 @@
+import { ethers } from "ethers";
+
+import {
+  AppInstance,
+  AppInterface,
+  AssetType,
+  computeStateHash,
+  expect,
+  Terms
+} from "./utils";
+
+import { ALICE, BOB } from "./constants";
+
+// HELPER DATA
+const TIMEOUT = 30;
```

Comment:
What is this timeout referring to? In the context of contracts and an async testing suite, it can be confusing to infer.